tracee/consts.go

```diff
@@ -28,6 +28,7 @@ const (
 	ACCESS_MODE_T ArgType = 20
 	PTRACE_REQ_T  ArgType = 21
 	PRCTL_OPT_T   ArgType = 22
+	ALERT_T       ArgType = 23
```

Comment:
I don't think we should use one arg type for all security alerts. it depends on my other comment about joining both memory alerts into one, if we do that then we can call this MPROT_ALERT_T or something similar

Reply:
Why adding too many argument types? The alert_t only encodes some message in most of the cases, and I don't see a reason now to create new special alerts